### ISSUE TITLE
Date of birth value returned with different format (Enhancement)

### ISSUE THREAD

**Issue by abomadi** [cloudscribe/cloudscribe#541]
Date of birth string format in user profile is different than admin panel user info, which is so confusing
- add DOB for site properties 
```
{
            "Key": "DateOfBirth",
            "Label": "Date of Birth",
            "EditPartialViewName": "FormItemDOBPartial",
            "ReadOnlyPartialViewName": "FormItemLabelPartial",
            "Tooltip": "",
            "DefaultValue": "",
            "CssClass": "dp",
            "IconCssClass": "far fa-calendar-alt",
            "IconOnLeft": "false",
            "VisibleOnRegistration": "false",
            "VisibleToUserOnProfile": "false",
            "EditableByUserOnProfile": "false",
            "VisibleOnAdminUserEdit": "true",
            "EditableOnAdminUserEdit": "true",
            "MaxLength": "30",
            "MaxLengthErrorMessage": "Date of Birth has a max length of 30 characters",
            "Required": "true",
            "RequiredErrorMessage": "Date of Birth is required yo!",
            "RegexValidationExpression": "",
            "RegexErrorMessage": "",
            "Options": []
          }
```

- Create new user inside admin panel > user management > new user and insert any value, date there appears as DD/MM/YYYY

- login using newly created user then go to profile > edit user info > the format here is different from the one was inserted before DD/MM/YY

**Comment by joeaudette** [comment]
I'm not having luck reproducing this problem. Can you clarify the culture settings you are using?

**Comment by joeaudette** [comment]
I did find some issues with the datepicker style that I have fixed in bootstrap4 views and piublished a new nuget.

But I did not reproduce the date format problem you describe.

Note that these settings for dob should all be false:

    "VisibleToUserOnProfile": "false",
            "EditableByUserOnProfile": "false",
            "VisibleOnAdminUserEdit": "false",
            "EditableOnAdminUserEdit": "false",

because although we use the kvp system to add dob to registration page and the new user admin page, dob is a real built in datetime field on siteuser so it is already on the user profile page and it is already in the admin manage for users. Threrefore those settings need to be false to avoid adding it twice on those pages.
